atomicapp/nulecule/base.py

```diff
@@ -148,12 +161,11 @@ def stop(self, provider_key=None, dryrun=False):
             None
         """
         provider_key, provider = self.get_provider(provider_key, dryrun)
-        # stop the Nulecule application
```

Comment:
I think that the code explains itself here when stopping (since it's the exact same as within `def run`